### PR TITLE
Update test eop merge

### PR DESCRIPTION
**Pull request by jgersak**
Add test for merging eopatches

### CODE REVIEW

**Comment by zigaLuksic** [comment]
I wouldn't create too advanced examples here, since this is mostly tested in `test_eodata_merge.py`. A simple test or two that it is correctly linked to `EOPatch` should suffice

**Comment by jgersak** [comment]
> I wouldn't create too advanced examples here, since this is mostly tested in test_eodata_merge.py. A simple test or two that it is correctly linked to EOPatch should suffice

Test case is significantly simplified, i wanted to write L529 less convoluted but kinda give up after a while ....

File: core/eolearn/tests/test_core_tasks.py [hunk 1]
```diff
@@ -481,9 +481,34 @@ def test_create_eopatch():
...
+        bbox=DUMMY_BBOX,
+        timestamps=dummy_timestamps,
+    )
 
-    task.execute(test_eopatch, test_eopatch, test_eopatch)
+    merged_patch = MergeEOPatchesTask(time_dependent_op="max", timeless_op="concatenate")(patch1, patch2)
+
+    expected_patch = EOPatch(
+        data={"bands": np.ones((2, 4, 4, 3), dtype=np.float32)},
+        mask_timeless={"LULC": np.arange(0, 32).reshape((16, 2), order="F").reshape(4, 4, 2)},
```
Comment (zigaLuksic) [inline core/eolearn/tests/test_core_tasks.py:504]: it is a bit difficult to understand, but the only other option I see is `np.concatenate([np.arange(0,16).reshape(...), ...])` which also feels a bit weird. So let's just leave it.

**Review by zigaLuksic — APPROVED** [review]

File: core/eolearn/tests/test_core_tasks.py (hunk 1 above)
Comment (jgersak) [reply core/eolearn/tests/test_core_tasks.py:504]: this solution would also demands that concatenation is done out side EOPatch init (i have tried it and like it even less)

**Pull request merged**